index.html

```diff
@@ -40,7 +40,7 @@
     <ul class="sidebar-nav">
         <a id="menu-close" href="#" class="btn btn-light btn-lg pull-right toggle"><i class="fa fa-times"></i></a>
         <li class="sidebar-brand">
-            <a href="#top">MalagaMakers</a>
+            <a href="#top">Malaga<wbr>Makers</a>

```

Comment:
I know what it means. OK, leave it, if you think that is a good opportunity to break the brand ;-)